modules/virt-creating-vm-import-cli.adoc

```diff
@@ -0,0 +1,182 @@
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: data-source-secret
+  labels:
+    app: containerized-data-importer
+type: Opaque
+data:
+  accessKeyId: "" <1>
+  secretKey:   "" <2>
+----
+<1> Specify the Base64-encoded key ID or user name.
+<2> Specify the Base64-encoded secret key or password.
+
+. Apply the `Secret` manifest by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f data-source-secret.yaml
+----
+
+. If the VM must communicate with servers that use self-signed certificates or certificates that are not signed by the system CA bundle, create a config map in the same namespace as the VM:
++
+[source,terminal]
+----
+$ oc create configmap tls-certs <1>
+  --from-file=</path/to/file/ca.pem> <2>
+----
+<1> Specify the config map name.
+<2> Specify the path to the CA certificate.
+
+. Edit the `VirtualMachine` manifest and save it as a `vm-fedora-datavolume.yaml` file:
++
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  creationTimestamp: null
+  labels:
+    kubevirt.io/vm: vm-fedora-datavolume
+  name: vm-fedora-datavolume <1>
+spec:
+  dataVolumeTemplates:
+  - metadata:
+      creationTimestamp: null
+      name: fedora-dv <2>
+    spec:
+      storage:
+        resources:
+          requests:
+            storage: 10Gi <3>
+        storageClassName: <storage_class> <4>
+      source:
+ifdef::url[]
+        http:
+          url: "https://mirror.arizona.edu/fedora/linux/releases/35/Cloud/x86_64/images/Fedora-Cloud-Base-35-1.2.x86_64.qcow2" <5>
+endif::[]
+ifdef::container-disks[]
+        registry:
+          url: "docker://kubevirt/fedora-cloud-container-disk-demo:latest" <5>
+endif::[]
+          secretRef: data-source-secret <6>
+          certConfigMap: tls-certs <7>
+    status: {}
+  running: true
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        kubevirt.io/vm: vm-fedora-datavolume
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: datavolumedisk1
+        machine:
+          type: ""
+        resources:
+          requests:
+            memory: 1.5Gi
+      terminationGracePeriodSeconds: 180
+      volumes:
+      - dataVolume:
+          name: fedora-dv
+        name: datavolumedisk1
+status: {}
+----
+<1> Specify the name of the VM.
+<2> Specify the name of the data volume.
+<3> Specify the size of the storage requested for the data volume.
+<4> Optional: If you do not specify a storage class, the default storage class is used.
+ifdef::url,container-disks[]
+<5> Specify the URL of the {data-source}.
+endif::[]
+<6> Optional: Specify the secret name if you created a secret for the {data-source} access credentials.
+<7> Optional: Specify a CA certificate config map.
+====
+
+. Create the VM by running the following command:
++
+[source,terminal]
+----
+$ oc create -f vm-fedora-datavolume.yaml
+----
```

Comment:
What's the difference between `oc create` and `oc apply`? You use create here and [apply above](https://github.com/openshift/openshift-docs/pull/64685/files#diff-662d8de3ca31ad88ab2f7eaf2c63ac7dd3774c3a75c1ae5bfc3a94d1145267d0R57)?

Reply:
[oc create](https://medium.com/@emmaliaocode/kubectl-create-vs-kubectl-apply-whats-the-differences-f6472f4c6c86) oc is just OpenShift kubectl + extras